boa/src/exec/tests.rs

```diff
@@ -247,3 +247,23 @@ fn test_short_circuit_evaluation() {
         "#;
     assert_eq!(exec(short_circuit_eval), String::from("1"));
 }
+
+#[test]
+fn test_console_assert() {
+    let multi_args = r#"
+        console.assert(12 > 13, "we should see that!", "that one too!", "that is not normal :/");
+    "#;
+    exec(multi_args);
+
```

Comment:
I would add some confirmation on what the output is.

Reply:
Is there a way to check side effects of function in unit tests in rust? It would be cool, but i don't know how to do it.

Reply:
Hmmm you mean the output to stdout/stderr? Not directly. As you say, this should output there, and that cannot be checked. In the future, we might have a Logger/Printer class that would be created with a writer for errors and so on.